     <script src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js" integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo=" crossorigin=""></script>
-    <script src="app.js?v=1.10"></script>
-    <script src="calendar.js?v=1.00"></script>
+    <script src="app.js?v=1.12"></script>
+    <script src="calendar.js?v=1.02"></script>
 </body>
 </html>
\ No newline at end of file
diff --git a/style.css b/style.css
index a52c821..e8c6e84 100644
--- a/style.css
+++ b/style.css
@@ -1,4 +1,4 @@
-/* Backroom public stylesheet v1.03 */
+/* Backroom public styles